batch/sql/estimated-current.sql

```diff
@@ -268,6 +309,7 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   `inst_coll` VARCHAR(255),
   `n_regions` INT DEFAULT NULL,
   `regions_bits_rep` BIGINT DEFAULT NULL,
+  `job_group_id` INT NOT NULL DEFAULT 0,
   PRIMARY KEY (`batch_id`, `job_id`),
```

Comment:
OK, so this is a really key and important aspect: `batch_id, job_id` is still the unique identifier of a job. Groups are merely some organizational structure atop all this. I think verbiage to this effect should make it into the RFC. What's the unique identifier for the relevant concepts. I think its:
1. Job: `batch_id, job_id`
2. Group: `batch_id, job_group_id`.
3. Batch: `batch_id`